content/platforms/kubernetes/kubernetes-architecture.md

```diff
@@ -1,44 +1,55 @@
 ---
 Title: Redis Labs Kubernetes Architecture
-description: 
+description: Redis Labs based its K8s architecture on the use of a standard
+  extension mechanism called an operator. The operator is service that
+  facilitates the specific needs of Redis Enterprise on Kubernetes and
+  guarantees the same set of enterprise features by translating them
+  into K8s-based pods and services whilst interacting with cluster events.
 weight: 20
 alwaysopen: false
 categories: ["Platforms"]
 aliases: /rs/concepts/kubernetes/redis-labs-kubernetes-architecture-overview
          /rs/concepts/kubernetes-architecture/
 ---
-Redis Labs bases its Kubernetes architecture on several vital concepts.
+Redis Labs bases its Kubernetes (K8s) architecture on several vital concepts.
 
 ## Layered architecture
 
-Kubernetes is an excellent orchestration tool, but it was not designed to deal with all the nuances associated with operating Redis Enterprise. Therefore, it can fail to react accurately to internal Redis Enterprise edge cases or failure conditions. Also, Kubernetes orchestration runs outside the Redis Cluster deployment and may fail to trigger failover events, for example, in split network scenarios.
+Kubernetes is an excellent and widely adopted orchestration platform. While it can provide the building blocks for a variety of applications, it was not designed to deal with all the nuances associated with operating Redis Enterprise. Therefore, without help, the Kubernetes cluster cannot react accurately to internal Redis Enterprise events, failure conditions, or user requests.
 
-To overcome these issues, Redis Labs created a layered architecture approach that splits responsibilities between operations Kubernetes does well, procedures Redis Enterprise Cluster excels at, and the processes both can orchestrate together. The figure below illustrated this layered orchestration architecture:
+The orchestration within a Kubernetes cluster runs outside the Redis Cluster deployment. As such, it is not aware of internal events and so can not respond to trigger failover events. For example, the Kubernetes cluster will not be aware of split network scenarios that affect Redis Enterprise clusters.
```

Comment:
Instead of addressing split-network scenario - K8s is responsible for the network and we can't necessarily address split network from within the cluster or Operator - so I would stick with the failover example.

Reply:
Split networks was in the original content. I can remove it.